nodeoperators/nodelist.json

```diff
@@ -2854,5 +2854,29 @@
         "Stake": 1000,
         "NetworkPubKey": "b722b50f98670fd55eb1958ddf9eafc7c926f43e79c03c86b8773ea38c4901729d38e06779a8bb904557a5559d51da0a44913fb6f47bea854bb54bd2b9bb122f",
         "StakingPubKey": "b8329dbafbd9f587bcffb98e5227c1ce7c80429329158b0c346bdbeebc250e47f52446eec0128b4e3bb19d3094c401f20f80936406e5d813d1a87db00e6a307b788c304929396ecdb1f8b73e7f996de807b68f74b5ce493296ed2d295557c831"
-      }
+    },
+    {
+        "Role": "consensus",
+        "Address": "p2p.97315cc5-894c-4228-96de-0f907d8aadd8.flow.bison.run:3569",
```

Comment:
waiting on BT to confirm. Will approve PR as soon as I hear from them.

Reply:
OK BT confirmed - there was [bug](https://discord.com/channels/613813861610684416/691791247353970769/836754982006554665) on their end which resulted in the 3659 port in the contract.
The 3569 in the network address is correct.